### PR TITLE
Multifile cli

### PR DESCRIPTION
**Pull request by slingload**
Pull request for `prospector` branch that adds multiple file support for Issue #20  

1) Added a for loop to cli in `__main__.py` to handle multiple files from the command line. The only change was renaming `@click.argument` to 'files' and adding the `nargs=-1` argument. 

2) Updated `tests/test_cli.py` to incorporate new functionality and it is passing according to pytest. Instead of adding a new file to `tests/resources/` , I just used the pre-existing `copyright.txt` file there. To get the test to pass I had to run the CLI first on both `shortstory.txt` and `copryright.txt` then appended the output to `cli-default.json` 

To run use: 
```sh 
$ prosegrinder file1 file2 file3
```

### CODE REVIEW

**Comment by davidlday** [comment]
@slingload - I'm working on moving linting to GitHub actions and ditch Codacy. Hopefully have all that completed and merged later today. So ignore the failing lint check for now - I'm fixing in my other PR.

File: prosegrinder/__main__.py [hunk 1]
```diff
@@ -5,22 +5,33 @@
...
+@click.option(
+    "-s",
+    "--save",
+    required=False,
+    type=click.File("w"),
+    help="File to save output to."
+)
+def cli(files, save, indent):
+    """setup the command line interface"""
+    for file in files:
```
Comment (davidlday) [inline prosegrinder/__main__.py:28]: Output should be valid JSON. Make a list and append each file's stats to it, something like:
```python
l = []
for file in files:
    filename = click.format_filename(file.name)
    text = file.read()
    p = Prose(text)
    d = {"filename": filename, "statistics": p.stats}
j = json.dumps(l, indent=indent)
if save:
    save.write(j)
else:
    click.echo(j)
```

The `cli-default.json` will need updated, too.

Comment (davidlday) [reply prosegrinder/__main__.py:28]: Good work, BTW!

**Review by davidlday — CHANGES_REQUESTED** [review]
Output should be valid, parsable JSON, but everything else looks great! Thank you!

File: prosegrinder/__main__.py (hunk 1 above)
Comment (slingload) [reply prosegrinder/__main__.py:29]: Good call. I updated it using your suggestion:
```python
def cli(files, save, indent):
    """Setup the command line interface"""
    processed_files = []
    for file in files:
        filename = click.format_filename(file.name)
        text = file.read()
        p = Prose(text)
        d = {"filename": filename, "statistics": p.stats}
        processed_files.append(d)
    j = json.dumps(processed_files, indent=indent)
    if save:
        save.write(j)
    else:
        click.echo(j)
```
This causes the output of `tests/resources/cli-default.json` to be an array and is valid JSON according to this [https://jsonlint.com/](url). 
I meant to add both changes as a single commit but goofed and forgot to make both updates simultaneously, so they are two separate commits. Also I tried to squash those other  intermediate commits in the commit history to clean it up, but couldn't figure out how to do that.

**Comment by davidlday** [comment]
Didn't mean to close this! Can you repoint it to main instead, pls?

**Pull request merged**